superduperdb/base/datalayer.py

```diff
@@ -846,6 +846,9 @@ def _create_children(self, component: Component, serialized: t.Dict):
                     'identifier': child,
                     'version': self.metadata.get_latest_version(child_type_id, child),
                 }
+                self.metadata.create_parent_child(
```

Comment:
why?

Reply:
Should we add a relation here, if we use string value of child?

Reply:
If not, we will miss the relationship between parent and child